### PR TITLE
RSWEB-7703: center body text on ias and vp patterns

### PR DESCRIPTION
**Pull request by writes**
Updates body text alignment for ias and value props pattern to be centered.

Goes with www PR https://github.rackspace.com/rswebteam/www/pull/1396

Example pages on local www
- http://local.rackspace.com:7080/cloud/orchestration
- http://local.rackspace.com:7080/node/7366

### CODE REVIEW

File: styleguide/_themes/derek/scss/patterns/cards/ias.scss [hunk 1]
```diff
@@ -39,8 +39,13 @@
 .iasWidget {
   display: block;
   padding: 20px 10px 0;
+  text-align: center;
   text-decoration: none;
   width: $full-width;
+
+  p {
```
Comment (writes) [inline styleguide/_themes/derek/scss/patterns/cards/ias.scss]: @bighappyface Yeah this is to center the body whether or not there is a <p> tag around the text for IaS and Value Props. When using Legacy HTML there are no tags set but when using the Full HTML/WYSIWYG there will be a <p> tag assigned automagically.

Legacy HTML - /cloud/orchestration
Full HTML - /node/7366

Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/patterns/cards/ias.scss]: 👍

Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/patterns/cards/ias.scss]: 👍  on the strategy. (ignore the review)

**Comment by writes** [comment]
Hey @bighappyface and @WidgetsBurritos I have updated the PR and added a corresponding www PR that removes unnecessary styles.

**Comment by bighappyface** [comment]
@writes one more thing - the `ticker` import is importing `fonts` again and causing double entries in `derek.css`:

https://github.com/rackerlabs/zoolander/blob/bc4c0ba0cfdc8db13ab8ec893de9aa76e7298b58/styleguide/_themes/derek/scss/_main.scss#L12

I vote to pull the ticker import out.

**Pull request merged**